src/ClockRGB.elm

```diff
@@ -0,0 +1,94 @@
+module Main exposing (Model, Msg(..), main)
```

Comment:
The name of the module needs to be the name of the file.
Also, don't expose `Msg`'s constructors & `main`.